### DISCUSSION TITLE
How to send file from webhook with a PHP Server

### DISCUSSION THREAD

**Discussion by elfata92** [vasani-arpit/WBOT#312] [Q&A]
Please give an example of sending file from method...

Because i

**Comment by vasani-arpit** [comment]
have you checked this https://github.com/vasani-arpit/WBOT/discussions/254 ?

**Reply by elfata92** [reply]
I have a webhook that I have connected to the database, and it's working fine now.  but in a case if the user sends a certain message,bi want to send a jpg file.  but I haven't found a way yet.  can you help me how to write the correct code,?
